src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,109 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int MIN_AGE = 18;
+    private static final int DEFAULT_AGE = 20;
+    private static final String DEFAULT_LOGIN = "admin";
+    private static final String DEFAULT_PASSWORD = "12345678";
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setAge(DEFAULT_AGE);
+        user.setLogin(DEFAULT_LOGIN);
+        user.setPassword(DEFAULT_PASSWORD);
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        assertEquals(user, registrationService.register(user));
+    }
+
+    @Test
+    void register_minAge_Ok() {
+        user.setAge(MIN_AGE);
+        assertEquals(user, registrationService.register(user));
+    }
+
+    @Test
+    void register_moreThenMinAge_Ok() {
+        user.setAge(MIN_AGE);
+        assertEquals(user, registrationService.register(user));
+    }
+
+    @Test
+    void register_nullAge_NotOk() {
+        user.setAge(0);
+        checkException(user);
+    }
```

Comment:
Null and zero aren't the same. Either the method naming or the age value are incorrect